package/base-files/files/lib/upgrade/common.sh

```diff
@@ -128,6 +128,21 @@ export_bootdevice() {
 					fi
 				done
 			;;
+			PARTUUID=[a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9]\
```

Comment:
Can this be simplified?

Reply:
@pprindeville Any suggestions here?

Reply:
I didn't have an issue with the way it was, frankly.

Reply:
I don't have a problem with it.